config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/util/gateway/BuilderUtils.java

```diff
@@ -76,10 +84,6 @@ static Element buildPropertiesElement(final Map<String, Object> properties, fina
             return emptyMap();
         }
 
-        if (!Objects.equals(propertiesElement.getNodeName(), PROPERTIES)) {
```

Comment:
We can probably keep this in here if we substitute for PROPERTIES for propertiesElementName